gitbook/tables/time-signatures.html

```diff
@@ -2361,75 +2361,213 @@ <h2 id="recommended-stylistic-alternates">Recommended stylistic alternates</h2>
 <tbody>
 <tr>
 <td style="text-align:center"><span class="bravura_large">&#xF440;</span></td>
-<td><strong>uniE080.ss04</strong><br><em>timeSig0Large</em><br>Time signature 0 (outside staff)</td>
+<td><strong>uniE080.salt01</strong><br><em>timeSig0Large</em><br>Time signature 0 (outside staff)</td>
 <td style="text-align:center"><span class="bravura_large">&#xF45D;</span></td>
-<td><strong>uniE080.ss01</strong><br><em>timeSig0Small</em><br>Time signature 0 (small staff)</td>
+<td><strong>uniE080.salt02</strong><br><em>timeSig0Small</em><br>Time signature 0 (small staff)</td>
 </tr>
 <tr>
+<td style="text-align:center"><span class="bravura_large">&#xF506;</span></td>
```

Comment:
The glyph values that have changed are no longer displaying correctly on my system, often appearing as clef glyphs instead.